.gitignore

```diff
@@ -32,3 +32,8 @@ Intermediate/*
 
 # Other
 .DS_Store
```

Comment:
Can you also change `Binaries/*` and `Intermediate/*` .gitignore entries to `plugin-dev/Binaries/*` and `plugin-dev/Intermediate/*` respectively please? I've missed that part for the previous PR